Formula/notion-md-gen.rb

```diff
@@ -0,0 +1,20 @@
+class NotionMdGen < Formula
+  desc "Convert notion content to markdown for static blog"
+  homepage "https://github.com/saltbo/notion-md-gen"
+  url "https://github.com/saltbo/notion-md-gen.git",
+      tag:      "v1.2.0",
+      revision: "6db355b7e63ba8b0b745989f84ff42b8c65f4637"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-ldflags",
+      "-s -w -X notion-md-gen/cmd.release=#{version} -X notion-md-gen/cmd.commit=#{Utils.git_head} -X notion-md-gen/cmd.repo=#{stable.url}",
+      *std_go_args,
+      "./main.go"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/notion-md-gen version 2>&1")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula